data-raw/simulate-data.R

```diff
@@ -151,6 +153,7 @@ create_fake_date <- function(n, from = "1977-01-01", to = lubridate::today()) {
 #' @examples
 #' create_padded_integer(5, 10)
 create_padded_integer <- function(n, length) {
+  set.seed(length)
```

Comment:
Is it an issue to set a seed inside a function? 
Is it an issue to set a seed here "locally" and also globally in the pipeline?